js/handlers/expenses.js

```diff
@@ -94,3 +94,21 @@ function expenseItem (expense) {
     <time date="${expense.date}">${(new Date(expense.date)).toDateString().slice(0, -5)}</time>
   </li>`
 }
+
+export function onSettleUpClick (event) {
+  event.preventDefault()
+  const target = event.target
+  const debt = JSON.parse(event.target.getAttribute("data"))
```

Comment:
Having to pass the data as an attribute is probably not ideal, but it is an interesting case.
Have you considered attaching an event handler directly to the HTML Element created by `makeDebt` in the other file? This would allow to access all the data for the expense